Revert commit 771004298, introduced in 3.14.4, that broke resume after
hibernate on yeeloong.  -lxoliva

diff --git arch/mips/power/hibernate.S arch/mips/power/hibernate.S
index 32a7c82..7e0277a 100644
--- arch/mips/power/hibernate.S
+++ arch/mips/power/hibernate.S
@@ -43,7 +43,6 @@ LEAF(swsusp_arch_resume)
 	bne t1, t3, 1b
 	PTR_L t0, PBE_NEXT(t0)
 	bnez t0, 0b
-	jal local_flush_tlb_all /* Avoid TLB mismatch after kernel resume */
 	PTR_LA t0, saved_regs
 	PTR_L ra, PT_R31(t0)
 	PTR_L sp, PT_R29(t0)
